synfig-studio/src/gui/renddesc.cpp

```diff
@@ -803,16 +815,18 @@ Widget_RendDesc::create_gamma_tab()
 		((Gtk::Label*)frame->get_label_widget())->set_markup(_("<b>Gamma Correction Settings</b>"));
 		panelBox->pack_start(*frame, Gtk::PACK_SHRINK);
 
-		frame->set_margin_start(6);
-		frame->set_margin_end(0);
-		frame->set_margin_top(6);
-		frame->set_margin_bottom(0);
 		gamma_frame = frame;
 		
 		Gtk::Grid *frameGrid = manage(new Gtk::Grid());
 		frame->add(*frameGrid);
 		frameGrid->set_row_spacing(10);
 		frameGrid->set_column_spacing(10);
+		frameGrid->set_margin_start(6);
+		frameGrid->set_margin_end(0);
+		frameGrid->set_margin_top(6);
+		frameGrid->set_margin_bottom(0);
```

Comment:
If set margin value is 0 no need to add them. It's redundant. By default, it was already 0.

Reply:
I will fix this asap, wherever I have done this ! :)

Reply:
I've made the required changes :)